src/App.tsx

```diff
@@ -1,19 +1,107 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useState, useEffect, useRef,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { TodosList } from './components/TodosList';
+import { getTodos, addTodo, removeTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { Navigation } from './components/Navigation';
+import { ErrorNotification } from './components/ErrorNotification';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<any>([]);
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [error, setError] = useState<boolean>(false);
+  const [status, setStatus] = useState('All');
+  const [textField, setTextField] = useState<string>('');
+  const [emptyInput, setEmptyInput] = useState<boolean>(false);
+  const [isAdding, setIsAdding] = useState<boolean>(true);
+
+  const changeStatusOnClick = (event: React.MouseEvent) => {
+    setStatus(event.currentTarget.innerHTML);
+  };
```

Comment:
this function wrapper is redundant, you can pass `setStatus` to `Navigation` component directly